trunk/SUAVE/Methods/Aerodynamics/AVL/create_avl_datastructure.py

```diff
@@ -313,14 +313,19 @@ def populate_wing_sections(avl_wing,suave_wing):
             origin.append( [[origin[i_segs][0][0] + dx , origin[i_segs][0][1] + dy, origin[i_segs][0][2] + dz]])               
 
     else:    
-        symm                  = avl_wing.symmetric
-        sweep                 = suave_wing.sweeps.quarter_chord
+        symm                  = avl_wing.symmetric  
         dihedral              = suave_wing.dihedral
         span                  = suave_wing.spans.projected
         semispan              = suave_wing.spans.projected * 0.5 * (2 - symm)
+        ar                    = suave_wing.aspect_ratio 
+        taper                 = suave_wing.taper      
         avl_wing.semispan     = semispan
-        origin                = suave_wing.origin[0]
-
+        origin                = suave_wing.origin[0] 
+        if suave_wing.sweeps.leading_edge is not None:  
+            sweep             = suave_wing.sweeps.leading_edge
+        else: 
+            sweep_qc          = suave_wing.sweeps.quarter_chord
```

Comment:
might as well write it back to the wing object for next time. Could help in debugging to know which value was used.

Reply:
We also have the convert sweep function. I would prefer you use that. But it's just one line.

Reply:
Ok I changed it to the planform function but still kept the conditional statement for speed